ee/pages/api/integrations/stripepayment/portal.ts

```diff
@@ -15,29 +15,7 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
       return;
     }
 
-    // Get user
-    const user = await prisma.user.findUnique({
-      where: {
-        id: session.user?.id,
-      },
-      select: {
-        email: true,
-        name: true,
-        metadata: true,
-      },
-    });
-
-    if (!user?.email)
-      return res.status(404).json({
-        message: "User email not found",
-      });
-
-    const customerId = await getStripeCustomerId(user);
-
-    if (!customerId)
-      return res.status(404).json({
-        message: "Stripe customer id not found",
-      });
+    const customerId = await getStripeCustomerFromUser(session.user.id);
 
     const return_url = `${process.env.BASE_URL}/settings/billing`;
     const stripeSession = await stripe.billingPortal.sessions.create({
```

Comment:
`customer: customerId,` throws a type error, which likely causes some side effects given the types come from stripe and are likely up to date, no?
<img width="999" alt="image" src="https://user-images.githubusercontent.com/8019099/152697278-6bbf477e-3be7-4ffb-af78-ed98a1c3beb1.png">

Reply:
looking at git-blame, i believe this was introduced earlier by @zomars

Reply:
fixed, was just expecting a non null value so I added a null check